internal/service/studies/main.go

```diff
@@ -17,6 +21,83 @@ func New() *Service {
 	}
 }
 
+func (s *Service) CreateStudy(userID uuid.UUID, studyData openapi.StudyCreateRequest) (*types.Study, error) {
+	if strings.TrimSpace(studyData.Title) == "" {
+		return nil, errors.New("study title is required")
+	}
+
+	if studyData.DataControllerOrganisation == nil || strings.TrimSpace(string(*studyData.DataControllerOrganisation)) == "" {
```

Comment:
if this is required then it can be a required field and we just check that it's not `""` here(?)

Reply:
field should already be required so it should throw an error if it doesn't have any content (have checked and it does do this), shouldn't the DataControllerOrganisation == "Other" not nil?

Reply:
doesn't look like it  – might have been missed in a refactor https://github.com/ucl-arc-tre/portal/blob/8075505a2369c8f5ee6c0619f4f1137838d3568b/api.web.yaml#L591-L593C11